cmd/snap/complete.go

```diff
@@ -508,7 +508,7 @@ func (snapshotID) Complete(match string) []flags.Completion {
 func (s snapshotID) ToUint() (uint64, error) {
 	setID, err := strconv.ParseUint((string)(s), 10, 64)
 	if err != nil {
-		return 0, fmt.Errorf(i18n.G("invalid argument for set id: expected a non-negative integer argument"))
+		return 0, fmt.Errorf(i18n.G("invalid argument for set id: expected a non-negative integer argument (see 'snap help saved')"))
```

Comment:
I think this message can be improved further by changing 'set' to 'snapshot set' :-)